src/components/Fund/FundPane.js

```diff
@@ -67,7 +67,7 @@ class FundPane extends Component {
     return (
       <form id="form-fund">
         <Pane defaultWidth="100%" firstMenu={firstMenu} lastMenu={lastMenu} paneTitle={paneTitle}>
-          <FundForm {...this.props} {...this.props} />
+          {/* <FundForm {...this.props} {...this.props} /> */}
```

Comment:
Commented for now base on a new structure. The form will be updated in [UIF-91](https://issues.folio.org/browse/UIF-91)